widgets/ui/src/main/resources/PhenoTips/CarePathways.xml

```diff
@@ -36,8 +36,1736 @@
+        #end
+        #set ($discard = $storedDatum.put("count", $q.number))
+        #set ($discard = $categoryItems.add($storedDatum))
+      #end
+    #end
+  #end
+#end
+##
+##
+#macro (__displayChildren $term $childrenList $question $storedValues)
+  #if ($childrenList.size() &gt; 0)
+    #set ($macro.accumulated = "")
+    #foreach($item in $childrenList)
+      #set ($childrenHtml = "#__displayChildren($item, $services.vocabularies.care-pathways.search({'is_a': $item.id}, {'sort': 'nameSort asc'}))")
+      #if ("$!{childrenHtml}" != "")
+        #set ($itemName = $stringtool.capitalize("$!{item.name}"))
+        #if ($stringtool.trim("$!{childrenHtml}") != "")
+          #set ($otherKey = $stringtool.join([$item.id, '_other'], ''))##
+          #set ($macro.accumulated = "${macro.accumulated}&lt;li class='term-entry'&gt;&lt;input class='term-id' type='hidden' value='$!{item.id}'&gt;&lt;label&gt;&lt;span id='cp-expand-tool-${item.id}' class='expand-tool fa fa-caret-right'&gt;&lt;/span&gt;&lt;span class='term-name'&gt;$!{itemName}&lt;/span&gt;&lt;/label&gt;&lt;ul&gt;${childrenHtml}&lt;li class='term-entry custom'&gt;&lt;input class='parent-id' type='hidden' value='$!{item.id}'&gt;&lt;label for='cp-add-tool-${question}${variantStub}-${item.id}_other'&gt;&lt;input id='cp-add-tool-${question}${variantStub}-${item.id}_other' class='add-tool' type='checkbox' #if($storedValues.containsKey($otherKey))checked#{end}&gt;&lt;span class='term-name'&gt;Other&lt;/span&gt;&lt;/label&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/li&gt;")
+        #else
+          #set ($macro.accumulated = "${macro.accumulated}&lt;li class='term-entry'&gt;&lt;input class='term-id' type='hidden' value='$!{item.id}'&gt;&lt;label for='cp-add-tool-${question}${variantStub}-${item.id}'&gt;&lt;input id='cp-add-tool-${question}${variantStub}-${item.id}' name='$!{item.id}' class='add-tool' type='checkbox' #if($storedValues.containsKey($item.id))checked#{end}&gt;&lt;span class='term-name'&gt;$!{itemName}&lt;/span&gt;&lt;/label&gt;&lt;/li&gt;")
+        #end
+      #end
+    #end
+    $macro.accumulated
+  #end
+#end
+##
+##
+#macro (__displayCategories $rootNode $rootChildren $question $storedValues)
+  #set ($otherId = $stringtool.join([$rootNode.id, '_other'], ''))##
+  &lt;ul&gt;
+    #__displayChildren($rootNode, $rootChildren, $question, $storedValues)
+    &lt;li class='term-entry custom'&gt;&lt;input class='parent-id' type='hidden' value='$!{rootNode.id}'&gt;&lt;label for='cp-add-tool-${question}${variantStub}-${rootNode.id}_other'&gt;&lt;input id='cp-add-tool-${question}${variantStub}-${rootNode.id}_other' class='add-tool' type='checkbox' #if($storedValues.containsKey($otherId))checked#{end}&gt;&lt;span class='term-name'&gt;Other&lt;/span&gt;&lt;/label&gt;&lt;/li&gt;
+  &lt;ul&gt;
+#end
+##
+##
+#macro (__displayModule $module $submodule $question $variantNumber $storedValues)
+  #set ($test = "CP:1")
+  #set ($care = "CP:2")
+  #set ($genetic = "CP:4")
+  #set ($diagnostic = "CP:68")
```

Comment:
Are you confident this will remain as `68`? Seems a pretty random number... Maybe move it up as one of the first terms?

Actually, I would use a more complicated pattern for these terms:

- `CP-0XXX` for important categories, like `CP1` and `CP2` are now
- `CP-1XXX` for test terms
- `CP-2XXX` for care terms
- `CP-3XXX` for family care terms

Is that feasible?

Reply:
Fixed.